x-pack/plugins/infra/public/metrics_overview_fetchers.test.ts

```diff
@@ -66,13 +66,14 @@ describe('Metrics UI Observability Homepage Functions', () => {
           start: 'now-15m',
           end: 'now',
         },
-        bucketSize,
+        bucketSize: 300,
+        intervalString,
       });
       expect(core.http.post).toHaveBeenCalledTimes(1);
       expect(core.http.post).toHaveBeenCalledWith('/api/metrics/overview/top', {
         body: JSON.stringify({
           sourceId: 'default',
-          bucketSize,
+          bucketSize: intervalString,
```

Comment:
This doesn't look right. `bucketSize` is a number and `intervalString` a string

Reply:
Ah.. I guess the metric api is out of out control...

Reply:
yes, that's why I ended up doing like this.